sources/SessionView.m

```diff
@@ -164,6 +164,8 @@ - (void)setDimmed:(BOOL)isDimmed {
     } else {
         _dim = isDimmed;
     }
+    
+    [[[self session] tab] recheckBlur];
```

Comment:
Is this really needed? It only has two callers, `setDimmingForSession:` (which already calls `recheckBlur`) and `-[PTYTab initWithSession:]`. It could be moved into `initWithSession:`, at least.